backend/routers/monthly_dumps.py

```
@@ -0,0 +1,138 @@
+    
+    # Auth check: Ensure user is creating for themselves
+    if current_user.user.id != user_id:
+        logger.warning(f"Unauthorized dump creation: {current_user.user.id} tried for {user_id}")
+        raise HTTPException(status_code=403, detail="Not authorized to create dump for this user")
+
+    month = normalize_month(payload.month)
+    timezone_name = payload.timezone or "UTC"
+    bucket_name = "monthly_dumps"
+    force = bool(payload.force)
+    
+    logger.info(
```

Ruff (0.15.10)

[error] Too many arguments for `logging` format string

(PLE1205)